src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -299,6 +299,7 @@ public enum CaseEvent {
     SET_LIP_RESPONDENT_RESPONSE_DEADLINE(CAMUNDA),
     NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
     NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
+    CREATE_SERVICE_REQUEST_CUI(CAMUNDA),
```

Comment:
can we make it more specific like CREATE_SERVICE_REQUEST_CUI_CLAIM_ISSUE

Reply:
Thanks @dharmendrak, I have updated the event name as you suggest.